doc/developer-guide/api/functions/TSContScheduleEvery.en.rst

```diff
@@ -33,19 +33,20 @@ Synopsis
 Description
 ===========
 
-Schedules :arg:`contp` to periodically run every :arg:`delay` milliseconds in the future.
-This is approximate. The delay will be at least :arg:`delay` but possibly more.
-Resolutions finer than roughly 5 milliseconds will not be effective. :arg:`contp` is
-required to have a mutex, which is provided to :func:`TSContCreate`.
+Schedules :arg:`contp` to run :arg:`every` milliseconds, on a random thread that belongs to
+:arg:`tp`. The :arg:`every` is an approximation, meaning it will be at least :arg:`every`
+milliseconds but possibly more. Resolutions finer than roughly 5 milliseconds will not be
```

Comment:
Hmm is the resolution really that consistent across different flavors of Linux, and FreeBSD and macOS?

Reply:
I'd assume so since this is in the docs way back before I ever touched anything.